.connector-store/meta.json

```diff
@@ -15,7 +20,231 @@
+          },
+          {
+              "name":"createRatio",
+              "description":"Create a Ratio datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createReference",
+              "description":"Create a Reference datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createRelatedArtifact",
+              "description":"Create a RelatedArtifact datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createSampledData",
+              "description":"Create a SampledData datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createSignature",
+              "description":"Create a Signature datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createSimpleQuantity",
+              "description":"Create a SimpleQuantity datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createTiming",
+              "description":"Create a Timing datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createTriggerDefinition",
+              "description":"Create a TriggerDefinition datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"createUsageContext",
+              "description":"Create a UsageContext datatype.",
+              "isHidden":false
+          },
+          {
+              "name":"evaluateFHIRPath",
+              "description":"Evaluate a FHIRPath expression.",
+              "isHidden":false
+          },
+          {
+              "name":"serialize",
+              "description":"Serialize a resource to a specific format.",
+              "isHidden":false
+          },
+          {
+              "name":"validate",
+              "description":"Validate a resource.",
+              "isHidden":false
+          },
+          {
+              "name":"setBundleIdentifier",
+              "description":"Set the identifier for the Bundle.",
+              "isHidden":false
+          },
+          {
+              "name":"setBundleType",
+              "description":"Set the type for the Bundle.",
+              "isHidden":false
+          },
+          {
+              "name":"setBundleTimestamp",
+              "description":"Set the timestamp for the Bundle.",
+              "isHidden":false
+          },
+          {
+              "name":"setBundleTotal",
+              "description":"Set the total for the Bundle.",
+              "isHidden":false
+          },
+          {
+              "name":"setBundleSignature",
+              "description":"Set the signature for the Bundle.",
+              "isHidden":false
+          }
+      ],
+      "connections": [
+          {
+              "name": "fhirbase",
+              "description": "Connection for accessing FHIRBase repository."
+          }
+      ],
```

Comment:
Since this is a module, shouldn't this be an empty array?

Reply:
Fixed in fb7601ae13153b814728d727fbf46774abc51338